Sprint-3/3-stretch/creditCard-validator.js

```diff
@@ -0,0 +1,37 @@
+function creditCardValidator(cardNumber) {
+  // Remove all - and spaces from the input
+  const sanitized = cardNumber.replace(/[-\s]/g, "");
+
+  //check if the length of the sanitized input is 16
+  if (sanitized.length !== 16) {
+    return false;
+  }
+  // check if the sanitized input contains only digits
+  if (!/^\d{16}$/.test(sanitized)) {
```

Comment:
By putting a `{16}` here, you're checking the length again. This means we're checking the length twice.

This is fine, but if we wanted to change the expected length of a credit card, we'd need to change both.

Can you think how to avoid this duplication?